exercises/modules/modules2.rs

```diff
@@ -1,11 +1,15 @@
 // modules2.rs
+// You can bring module paths into scopes and provide new names for them with the
+// 'use' and 'as' keyword. Fix these 'use' statments to make the code compile.
 // Make me compile! Execute `rustlings hint modules2` for hints :)
 
 // I AM NOT DONE
 
 mod delicious_snacks {
-    use self::fruits::PEAR as fruit;
-    use self::veggies::CUCUMBER as veggie;
+
+    // TODO: Fix these use statments
+    use self::fruits::PEAR as ???
```

Comment:
IMO, the hint for this needs updating as well, since it says `It's almost there except for one keyword missing for each constant.`, which is not true anymore.